File: cms_server/cms_server/Entities/ConsultAllergyEntity .cs
using System;

namespace cms_server.Entities
{
    public class ConsultAllergyEntity
    {
        public int? cr_allergy_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string substance { get; set; }
        public string reaction { get; set; }
        public DateTime? first_occur { get; set; }
        public string notes { get; set; }
        public string is_active { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}

File: cms_web/src/Services/Entities/ConsultAllergyEntity.ts
interface ConsultAllergyEntity {
  cr_allergy_pk?: number;
  consult_req_pk?: string;
  substance: string;
  reaction: string;
  first_occur: string;
  notes: string;
  is_active?: "y" | "n";
  encoded_at?: string | Date;
  encoder_pk?: string;
}

export interface ConsultAllergyTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultAllergyEntity>;
}

export default ConsultAllergyEntity;
